Herbert/Framework/Enqueue.php

```diff
@@ -66,13 +66,15 @@ public function buildInclude($attrs, $footer)
 //            $attrs['src'] = ltrim($attrs['src'], '/');
 //        }
 
+        if (!isset($attrs['dep'])) $attrs['dep'] = [];
```

Comment:
I'd either new line this and use curlies or, on L77 use `array_get($attrs, 'dep', [])` instead.